shared/templates/sysctl/bash.template

```diff
@@ -6,7 +6,11 @@
 
 # Comment out any occurrences of {{{ SYSCTLVAR }}} from /etc/sysctl.d/*.conf files
 {{% if product not in [ "ol7", "ol8", "ol9", "rhcos4", "rhel7", "rhel8", "rhel9"] %}}
+{{% if product in [ "sle12", "sle15"] %}}
```

Comment:
for the purpose of readability it would be better to avoid the nested conditions and instead use if-elif-else tree